memconfig.lock

```diff
@@ -21,19 +21,19 @@
                 "params": [
                   {
                     "name": "url",
-                    "description": "The URL to send the POST request to",
```

Comment:
@aranajhonny when you get a chance, update your `http` and I think most of these changes will go away.